cypress/e2e/pages/homePage/testsForCanonicalOnly.js

```diff
@@ -39,6 +41,46 @@ export default ({ service }) => {
       });
     });
   });
+
+  describe('Portrait Video Curations', () => {
+    beforeEach(() => {
+      cy.viewport(600, 800);
+    });
+    // open and close could be 2 separate tests, but would be largely duplicated or tests reliant on the previous ones state for the modal to be open
+    it('should open the portrait video modal when a promo is clicked and close it when the modal close button is clicked', () => {
```

Comment:
Do we need to do a separate test for clicking on the background (and not the modal close button?)

Reply:
As that has been newly added as a feature, yes I'll try adding that now